<HTML>
<HEAD>
<meta charset="UTF-8">
<title>TransactionVerificationException.<init> - corda</title>
<link rel="stylesheet" href="../../../style.css">
</HEAD>
<BODY>
<a href="../../index.html">corda</a>&nbsp;/&nbsp;<a href="../index.html">net.corda.core.contracts</a>&nbsp;/&nbsp;<a href="index.html">TransactionVerificationException</a>&nbsp;/&nbsp;<a href="./-init-.html">&lt;init&gt;</a><br/>
<br/>
<h1>&lt;init&gt;</h1>
<a name="net.corda.core.contracts.TransactionVerificationException$&lt;init&gt;(net.corda.core.crypto.SecureHash, kotlin.String, kotlin.Throwable)"></a>
<code><span class="identifier">TransactionVerificationException</span><span class="symbol">(</span><span class="identifier" id="net.corda.core.contracts.TransactionVerificationException$<init>(net.corda.core.crypto.SecureHash, kotlin.String, kotlin.Throwable)/txId">txId</span><span class="symbol">:</span>&nbsp;<a href="../../net.corda.core.crypto/-secure-hash/index.html"><span class="identifier">SecureHash</span></a><span class="symbol">, </span><span class="identifier" id="net.corda.core.contracts.TransactionVerificationException$<init>(net.corda.core.crypto.SecureHash, kotlin.String, kotlin.Throwable)/message">message</span><span class="symbol">:</span>&nbsp;<a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-string/index.html"><span class="identifier">String</span></a><span class="symbol">, </span><span class="identifier" id="net.corda.core.contracts.TransactionVerificationException$<init>(net.corda.core.crypto.SecureHash, kotlin.String, kotlin.Throwable)/cause">cause</span><span class="symbol">:</span>&nbsp;<a href="https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-throwable/index.html"><span class="identifier">Throwable</span></a><span class="symbol">?</span><span class="symbol">)</span></code>
<p>Indicates that some aspect of the transaction named by <a href="tx-id.html">txId</a> violates the platform rules. The exact type of failure
is expressed using a subclass. TransactionVerificationException is a <a href="../../net.corda.core.flows/-flow-exception/index.html">FlowException</a> and thus when thrown inside
a flow, the details of the failure will be serialised, propagated to the peer and rethrown.</p>
</BODY>
</HTML>
